docker/management-ui/Dockerfile-dev

```diff
@@ -13,14 +13,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-FROM nginx:1.23-alpine
+FROM graviteeio/nginx:1.22
```

Comment:
gio tag not consistent with the nginx tag used to build this [base image](https://github.com/gravitee-io/gravitee-docker/blob/master/images/nginx/Dockerfile#L13) 

Should be graviteeio/nginx:1.23

Reply:
upgraded